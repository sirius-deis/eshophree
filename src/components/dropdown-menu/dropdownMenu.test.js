import { render, screen, fireEvent } from '@testing-library/react';

import DropdownMenu from './dropdownMenu';

describe('DropdownMenu component', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <DropdownMenu>
        <button>button #1</button>
        <button>button #2</button>
      </DropdownMenu>,
    );
    expect(container).toMatchSnapshot();
  });
  it('should render content', () => {
    render(
      <DropdownMenu>
        <button>button #1</button>
        <button>button #2</button>
      </DropdownMenu>,
    );
    const content = screen.queryAllByRole('button');
    expect(content[0]).toHaveTextContent('button #1');
    expect(content[1]).toHaveTextContent('button #2');
    expect(content.length).toBe(2);
  });
});
