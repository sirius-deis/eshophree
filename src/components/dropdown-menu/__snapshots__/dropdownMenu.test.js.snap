// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownMenu component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB kTYWeF"
  >
    <button>
      button #1
    </button>
    <button>
      button #2
    </button>
  </div>
</div>
`;
